### PR TITLE
feat(release): add semantic release prod plugin

### PR DESCRIPTION
**Pull request by OlegDO**

### CODE REVIEW

File: src/release.config.mjs [hunk 1]
```diff
@@ -21,11 +27,13 @@ export default {
     [
       '@semantic-release/exec',
       {
-        publishCmd:
-          'zip -r build.zip lib' +
-          " && echo '${nextRelease.version}' > .version",
+        analyzeCommitsCmd: 'echo "minor"',
+        generateNotesCmd: 'echo "chore: bump version"'
```
Comment (MatthewPattell) [inline src/release.config.mjs]: why you change standard config???

File: src/release.config.mjs [hunk 2]
```diff
@@ -41,5 +49,14 @@ export default {
...
     ],
+    [
+      '@semantic-release/github',
+      {
+        labels: false,
+        releasedLabels: false,
+        successComment: false,
+        assets: [{ path: 'build.zip', label: 'Build-${nextRelease.gitTag}' }],
+      },
+    ],
```
Comment (MatthewPattell) [inline src/release.config.mjs]: ?????

File: src/release.config.mjs [hunk 3]
```diff
@@ -21,11 +27,13 @@ export default {
...
       {
-        publishCmd:
-          'zip -r build.zip lib' +
-          " && echo '${nextRelease.version}' > .version",
+        analyzeCommitsCmd: 'echo "minor"',
+        generateNotesCmd: 'echo "chore: bump version"'
       },
     ],
+    ...(shouldProcessProdPlugin && [
+      path.join(__dirname, 'plugins', 'release', 'index.js'),
```
Comment (MatthewPattell) [inline src/release.config.mjs]: better way?

**Review by MatthewPattell — CHANGES_REQUESTED** [review]

File: src/release.config.mjs [hunk 4]
```diff
@@ -21,11 +24,16 @@ export default {
     [
       '@semantic-release/exec',
       {
+        analyzeCommitsCmd: 'echo "minor"',
+        generateNotesCmd: 'echo "chore: bump version"',
```
Comment (MatthewPattell) [inline src/release.config.mjs]: you can also move this to `plugins.RELEASE`.

Comment (MatthewPattell) [reply src/release.config.mjs]: plugins.RELEASE:

[
 { '@semantic-release/exec', { .... } },
 { your plugin }
]

Comment (OlegDO) [reply src/release.config.mjs]: got it

File: src/plugins/index.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+const RELEASE = path.join(__dirname, 'release', 'index.js')
+
+const plugins = {
+  RELEASE,
+}
```
Comment (MatthewPattell) [inline src/plugins/index.js]:
```suggestion
const plugins = {
  [
      '@semantic-release/exec',
      {
            analyzeCommitsCmd: 'echo "minor"',
            generateNotesCmd: 'echo "chore: bump version"'
      },
    ],
  RELEASE,
}
```

File: src/release.config.mjs [hunk 6]
```diff
@@ -24,8 +27,14 @@ export default {
         publishCmd:
           'zip -r build.zip lib' +
           " && echo '${nextRelease.version}' > .version",
+        ...(shouldPerformProdPlugin ?
+          {
+            analyzeCommitsCmd: 'echo "minor"',
+            generateNotesCmd: 'echo "chore: bump version"'
+          } : {})
```
Comment (MatthewPattell) [inline src/release.config.mjs]:
```suggestion
```

**Review by MatthewPattell — CHANGES_REQUESTED** [review]

File: src/plugins/release/index.js [hunk 7]
```diff
@@ -0,0 +1,179 @@
+const fs = require('fs');
+const path = require('path');
+
+const parentDirectory = '..'; // Microservices folder
+const rootMicroservicesDirectory = path.join('..', '..')// Root microservices folder
```
Comment (MatthewPattell) [inline src/plugins/release/index.js]:
```suggestion
const rootMicroservicesDirectory = path.join('..', '..'); // Root microservices folder
```

File: src/plugins/index.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+const plugins = {
+  RELEASE: path.join(__dirname, 'release', 'index.js'),
+  RELEASE_CMD: [
+    '@semantic-release/exec',
+    {
+      analyzeCommitsCmd: 'echo "minor"',
+      generateNotesCmd: 'echo "chore: bump version"'
+    },
+  ],
+}
```
Comment (MatthewPattell) [inline src/plugins/index.js]:
```suggestion
const plugins = [
 [
   path.join(__dirname, 'release', 'index.js'),
  ],
  [
    '@semantic-release/exec',
    {
      analyzeCommitsCmd: 'echo "minor"',
      generateNotesCmd: 'echo "chore: bump version"'
    },
  ],
]
```

File: src/release.config.mjs [hunk 9]
```diff
@@ -26,6 +29,8 @@ export default {
           " && echo '${nextRelease.version}' > .version",
       },
     ],
+    ...(shouldPerformProdPlugin ? [plugins.RELEASE] : []),
+    ...(shouldPerformProdPlugin ? [plugins.RELEASE_CMD] : []),
```
Comment (MatthewPattell) [inline src/release.config.mjs]:
```suggestion
    ...(shouldPerformProdPlugin ? [releasePlugins] : []),
```

File: src/release.config.mjs [hunk 10]
```diff
@@ -1,4 +1,7 @@
+import plugins from "./plugins/index.js";
```
Comment (MatthewPattell) [inline src/release.config.mjs]:
```suggestion
import releasePlugins from "./plugins/index.js";
```

File: src/plugins/index.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
+const path = require('path');
```
Comment (MatthewPattell) [inline src/plugins/index.js:1]: why you use require instead import? ESM?

**Review by MatthewPattell — CHANGES_REQUESTED** [review]

File: src/plugins/index.js (hunk 11 above)
Comment (OlegDO) [reply src/plugins/index.js:1]: It cause errors even if I will define this file as .mjs with the es6 module i/e

**Review by MatthewPattell — APPROVED** [review]

**Pull request merged**